example/AcceleratedGaussianBlurring.ipynb

```diff
@@ -0,0 +1,343 @@
+      "text/plain": [
+       "<Figure size 720x288 with 3 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Blurring by FFT Convolution\n",
+    "tstart = time.time()\n",
+    "fourier_blurred_image = itk.fft_discrete_gaussian_image_filter(mri_image, **blur_args)\n",
```

Comment:
Perhaps an error in my configuration, but even with building ITK from scratch I am getting:
```
AttributeError: module 'itk' has no attribute 'fft_discrete_gaussian_image_filter'
```

Reply:
`itk.FFTDiscreteGaussianImageFilter` is [wrapped](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Filtering/Smoothing/wrapping/itkFFTDiscreteGaussianImageFilter.wrap) in the ITK main branch. Could you please add what ITK version you are using? I've also found it useful to check that I am referencing the expected ITK library directory with `itk.__path__`.

Reply:
When I launch python from my shell prompt, I get `itk.__version__ = 5.3.0` and `itk.__path__` does point to the wrapped itk directory in my build directory.  However, I am doing something wrong because my jupyter lab is not recognizing my Python virtual environment despite my `ipython kernel install` command.  I'll keep trying for a little longer.

Reply:
Understood. Is the wrapping for `itk.FFTDiscreteGaussianImageFilter` present when running your shell in your virtual environment?

Reply:
`itk.fft_discrete_gaussian_image_filter` is present from my shell-based virtual environment.  It's the transition to jupyter lab that is proving to be the problem.